.changeset/modern-apples-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hey-api/openapi-ts': minor
```

Comment:
Also I didn't notice earlier but this should be a patch release, it's not a new feature!